examples/flax/vision/requirements.txt

```diff
@@ -3,6 +3,6 @@ jaxlib>=0.1.59
 flax>=0.3.5
 optax>=0.0.8
 -f https://download.pytorch.org/whl/torch_stable.html
-torch==1.9.0+cpu 
+torch==1.11.0+cpu
 -f https://download.pytorch.org/whl/torch_stable.html
-torchvision==0.10.0+cpu
\ No newline at end of file
+torchvision==0.12.0+cpu
```

Comment:
Perfect! torchvision v0.12.0 is compatible with torch 1.11.0 as per https://pytorch.org/get-started/previous-versions/#v1110